libraries/aws_conn.rb

```diff
@@ -28,4 +28,12 @@ def iam_resource
   def iam_client
     @iam_client ||= Aws::IAM::Client.new
   end
+
+  def sts_client
+    @sts_client ||= Aws::STS::Client.new
+  end
+
+  def account_id
+    sts_client.get_caller_identity[:account]
```

Comment:
Because a user from account A can issue commands against account B--by assuming a role in account B [1]--I'm not clear on this documentation either way: in this case, which account ID would be returned?

I'm also a bit concerned about needing a different AWS service (STS) to answer an IAM question, but that's not a biggie.

Based on this, I have two alternatives to propose:
1. Verify the cross-account account ID lookup; if it works as desired, then use it to get the account ID.
2. Decide not to get/show the account ID.

What do you think?